packages/core/saved-objects/core-saved-objects-api-server-internal/src/lib/collect_multi_namespace_references.ts

```diff
@@ -7,19 +7,20 @@
  */
 
 import { isNotFoundFromUnsupportedServer } from '@kbn/core-elasticsearch-server-internal';
-import type { SavedObject } from '@kbn/core-saved-objects-server';
 import type {
   SavedObjectsCollectMultiNamespaceReferencesObject,
   SavedObjectsCollectMultiNamespaceReferencesOptions,
   SavedObjectsCollectMultiNamespaceReferencesResponse,
   SavedObjectReferenceWithContext,
 } from '@kbn/core-saved-objects-api-server';
 import {
-  AuditAction,
   type ISavedObjectsSecurityExtension,
   type ISavedObjectTypeRegistry,
+  type SavedObject,
 } from '@kbn/core-saved-objects-server';
-import { SavedObjectsErrorHelpers, SavedObjectsUtils } from '@kbn/core-saved-objects-utils-server';
+import { SavedObjectsUtils } from '@kbn/core-saved-objects-utils-server';
+import { SavedObjectsErrorHelpers } from '@kbn/core-saved-objects-common';
```

Comment:
Why was `SavedObjectsErrorHelpers` moved to a `common` package?

Reply:
That was an open question of mine (https://github.com/elastic/kibana/pull/148165#discussion_r1092151595). Thanks! I will find a more suitable place for it.